moped-editor/src/views/projects/projectView/ProjectPhaseForm.js

```diff
@@ -156,9 +155,9 @@ const ProjectPhaseForm = ({
       today.setHours(0, 0, 0, 0);
 
       setValue("phase_start", today);
-      return isCurrentPhase;
+      setValue("is_current_phase", isCurrentPhase, { shouldDirty: true });
```

Comment:
The `shouldDirty` option makes sure that the logic to disable the **Save** button stays intact since using `setValue` does not dirty the form by default which is a [TIL for me](https://react-hook-form.com/docs/useform/setvalue).